upload-agent/test.py

```diff
@@ -1,32 +1,246 @@
 from http.server import HTTPServer, BaseHTTPRequestHandler
 
+
+import os
+from time import sleep
+import unittest
+import threading
+import hashlib
+import shutil
+import subprocess
+import signal
+
 from io import BytesIO
 from base64 import b64decode
 
+TEST_DATA_DIR = "test_data"
 
-class SimpleHTTPRequestHandler(BaseHTTPRequestHandler):
+TEST_FILE_1_CORRECT = "correct_1.pcap"
 
-    # определяем метод `do_GET` 
-    def do_GET(self):
-        self.send_response(200)
-        self.send_header('Content-type', 'text/html')
-        self.end_headers()
-        self.wfile.write(b'Hello, world!')
+TEST_FILE_2_CORRECT = "correct_2.pcap"
+
+TEST_FILE_1_INCORRECT = "incorrect_1.pcap"
+
+TEST_DIR = "test_dir"
+
+SCRIPT_TO_RUN = "./upload-agent/main.py"
+
+STANDART_TIMEOUT = 1.5
+
+
+def get_file_hash(filename):
+    with open(filename, "rb") as file:
+        all_file = file.read()
+        return hashlib.md5(all_file).digest()
+
+
+def get_filepath_in_test_data_dir(filename: str) -> str:
```

Comment:
Длинные названия, сделай короче. типа такого
```suggestion
def test_file_path(filename: str) -> str:
```